core/src/trezor/ui/layouts/menu.py

```diff
@@ -116,3 +116,15 @@ async def confirm_with_menu(
             await show_menu(menu, raise_on_cancel)
         else:
             return result
+
+
+async def confirm_with_menu(
+    main: trezorui_api.LayoutObj[T],
+    menu: Menu,
+    br_name: str | None,
+    br_code: ButtonRequestType = ButtonRequestType.Other,
+    raise_on_cancel: ExceptionType = ActionCancelled,
+) -> None:
+    result = await interact_with_menu(main, menu, br_name, br_code, raise_on_cancel)
+    # use this function when the layout may only return CONFIRMED on success (or raise an exception)
```

Comment:
Can we edit this comment to mention that the concern is, for example **`BACK`** disrupting the `CONFIRMED` / `ActionCancelled` combo?

Also, this comment sounds more like it should be a docstring, no?